### ISSUE TITLE
Is it tested on Windows environment (Direct3D)

### ISSUE THREAD

**Issue by nidin** [kantedal/pathtracer-webgl2#1]
It is failing on my windows machine.
OS: Windows 10 Pro
Browser : Chrome, Firefox
CPU : Intel i7
GPU: RX 480 8GB

Reason: 
Unable to unroll loop, loop does not appear to terminate in a timely manner (161 iterations) or unrolled loop is too large

**Comment by kantedal** [comment]
No, I have only tested it on MacOS. But I will look into this, thanks for feedback!
